src/dscanner/main.d

```diff
@@ -348,6 +348,11 @@ Options:
         Prints the number of tokens in the given source files. If no files are
         specified, input is read from stdin.
 
+    --tokenDump <file>
+        Dump token information from the lexer. This option is mostly useful for
+        developing D-Scanner or its supporting libraries itself. You probably
+        dont't want to use this, and this feature may be removed in the future.
+
```

Comment:
I don't see why we should include this in the documentation at all

Reply:
I debated leaving this out, but it may be useful to someone. I figured that if it's in the tool (and not one of the dumb Easter eggs) it's better to document it.